src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -345,5 +399,49 @@ public static void MapToolTip(IViewHandler handler, IView view)
 				handler.ToPlatform().UpdateToolTip(tooltipContainer.ToolTip);
 #endif
 		}
+
+		void OnChangedContainerCollectionChanged(object? sender, EventArgs e) =>
+			UpdateValue(nameof(IDynamicContainerViewHandler.NeedsContainer));
+
+		class CollectionWithEvents : ICollection<string>
+		{
+			HashSet<string> properties = new();
```

Comment:
The collection is special, and knows when a new property has been added. When a new property is added, it will fire the whole container view code.

So if you are adding 2 props that add a container, no need to go through the whole process if the ContainerView is already set - just use it.